cpg_infra/abstraction/azure.py

```diff
@@ -270,8 +270,12 @@ def create_bucket(
         unique: bool = False,
         requester_pays: bool = False,
         versioning: bool = True,
+        autoclass: bool = False,
         project: str = None,
     ) -> Any:
+        # Autoclass is not supported on Azure yet.
```

Comment:
This will crash immediately, because there's no difference between the gcp and azure calls. I think it would be reasonable to ignore, or throw a warning, but assert will block the deploy.

Reply:
Oh, interesting. So we always create the buckets for all datasets on both GCP and Azure? I thought we'd want some sort of selector at the dataset level.

Reply:
There is a selector for the dataset level, but cpg-common is already spun up across both.

Reply:
Of course, good point.

Reply:
Changed this to a warning now.